### PR TITLE
Support AlmaLinux 8

### PR DESCRIPTION
**Pull request by EMH333**

### CODE REVIEW

File: spec/unit/recipes/server_spec.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
         expect { chef_run }.to_not raise_error
       end
 
-      if p[:platform] == 'centos'
+      if p[:platform] == 'centos' || p[:platform] == 'almalinux'
```
Comment (ramereth) [inline spec/unit/recipes/server_spec.rb:16]: I think a case statement might be better here.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
patch bump

**Comment by EMH333** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.2.1.  Have a nice day!

**Pull request merged**